.github/workflows/codeql-analysis.yml

```
@@ -24,6 +24,7 @@ jobs:
   analyze:
     name: Analyze
     runs-on: ubuntu-latest
+    if: startsWith( github.repository, 'mdn/' )
```

Let's be consistent with [workflows in other repos](https://github.com/mdn/content/blob/6fa75c683af3831c0e988a9270e3b870852dc9a1/.github/workflows/markdown-lint-fix.yml#L10):

```suggestion
    if: github.repository == 'mdn/interactive-examples'
```

---

sure, done :)